### PR TITLE
Update dev-bn branch with main branch

### PR DESCRIPTION
**Pull request by mitul3737**
This PR will update `dev-bn` branch with `main` branch.

I rebased `dev-bn` with main and handled conflicts.
Tracking isue: #410

### CODE REVIEW

File: content/bn/_index.md [hunk 1]
```diff
@@ -14,10 +14,9 @@ title: "ক্লাউড নেটিভ শব্দকোষ"
...
+ক্লাউড নেটিভ পরিভাষাটি CNCF মার্কেটিং কমিটি ও বিভিএস দ্বারা প্রবর্তন করা হয়েছিল এবং এতে [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/) , [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/) , [Katelin Ramer](https://www.linkedin.com/in/katelinramer/) , [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) এবং অন্যান্য অনেক অবদানকারীর অবদান অন্তর্ভুক্ত রয়েছে।
```
Comment (jihoon-seo) [inline content/bn/_index.md:17]: The English translation of this section with Google Translator is
> **The term Cloud Native was coined** by the CNCF Marketing Committee **and VVS** and includes contributions from Catherine Paganini, Chris Aniszczyk, Daniel Jones, Jason Morgan, Katelin Ramer, Mike Foster, Seokho Son and many others.

which is slightly different from the original English text:

> **The Cloud Native Glossary was initiated** by the CNCF Marketing Committee **(Business Value Subcommittee)** and includes contributions from [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), and many more contributors. **For a complete contributor list, please refer to [this GitHub page](https://github.com/cncf/glossary/graphs/contributors).**
> 
> **The Glossary is maintained by [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/), [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/), and [Seokho Son](https://www.linkedin.com/in/seokho-son/).**

@mitul3737 Could you check and update one more time? 😊

Comment (mitul3737) [reply content/bn/_index.md:17]: Yes, sure  @jihoon-seo  ☺️

Comment (mitul3737) [reply content/bn/_index.md:17]: Hey @jihoon-seo 
Will this be alright?
`ক্লাউড নেটিভ শব্দকোষটি CNCF মার্কেটিং কমিটি (বিজনেস ভ্যালু সাবকমিটি) দ্বারা সূচিত হয়েছিল এবং এতে [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/) , [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/) , [Katelin Ramer](https://www.linkedin.com/in/katelinramer/) , [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca) এবং আরও অনেক অবদানকারীর অবদান অন্তর্ভুক্ত রয়েছে। একটি সম্পূর্ণ অবদানকারী তালিকার জন্য, অনুগ্রহ করে এই [GitHub পৃষ্ঠাটি](https://github.com/cncf/glossary/graphs/contributors) দেখুন।


ক্লাউড নেটিভ ভোকাবুলারির বাংলা স্থানীয়করণের সূচনা [Bengali localization team](https://cloud-native.slack.com/archives/C02UG2WGXQQ) দ্বারা করা হয়েছে এবং এতে অন্তর্ভুক্ত রয়েছে [MD Shahriyar Al Mustakim Mitul](https://www.linkedin.com/in/md-shahriyar-al-mustakim-mitul-9084b31a0/), [MD Ikramul Kayes](https://www.linkedin.com/in/md-ikramul-kayes-753674214/), [Umme Abira Azmary](https://www.linkedin.com/in/umme-abira-azmary-68404a1bb/) এবং আরও অনেক অবদানকারী।`

**Comment by seokho-son** [comment]
Hi @jihoon-seo @mitul3737 

I hope to avoid including other commits (changing or translating contents) in this PR (PR for a branch sync) except resolving conflicts.
A commit to update contents itself needs to be handled in another PR.

So, please do not push other commits to this PR (PR for a branch sync) except resolving conflicts.

If this PR includes multiple commits to modify contents itself, this PR becomes very hard to be reviewed (checked) since this kind of PR includes existing commits from other authors. Commit messages will be hard to be managed. (we cannot use commit squashing method for merge)

(@jihoon-seo please review PRs related with branch sync in this way and guide l10n teams)

**Comment by mitul3737** [comment]
@seokho-son  @jihoon-seo Okay! Do let me know if any step is needed to be done 😊

**Comment by jihoon-seo** [comment]
The PR #879 supersedes this one. Let's close this.

**Pull request closed without merging**